### PR TITLE
148 minimize access for pure admin

### PR DESCRIPTION
**Pull request by EWLameijer**
This version should hopefully work well.

### CODE REVIEW

File: Documentation/CodeOverview/book_documentation.adoc [hunk 1]
```diff
@@ -1,6 +1,15 @@
 = Documentation
 :toc:
 
+== Ubiquitous Language
+- *attendance*: a student being present (or belatedly present) or absent on a lesson.
+- *group*: a group of students following a course at ITVitae, for example Java55
+- *lesson*: a session scheduled for a group, on a day, with a teacher. There is only one lesson per day, and the lesson can have only one teacher. A teacher cannot teach two lessons on a day.
```
Comment (MtePoele) [inline Documentation/CodeOverview/book_documentation.adoc:7]: There are usually multiple lessons on the same day. But each group can only have up to one lesson per day.

**Review by RaafiKuijper — APPROVED** [review]
No further comments

**Comment by MtePoele** [comment]
Possible bug: The same email adress can be used for more than one user.
Both for separate roles and same user.
Tested with 2 "Docent" and 1 "Studentbegeleider".

**Comment by MtePoele** [comment]
Possible bug:
If there is no attendance for a student, the percentages show as NaN%. This might need to say something like "nog geen aanwezigheids geregistreerd" or even just be blank after the =.
afbeelding

**Comment by MtePoele** [comment]
Bug: Deleted group with only classes in future doesn't free up the teacher.

Setup:
Make a group (in my case B)
Add future lessons (in my case Wubbo as teacher, random dates)
Delete group B

Result:
afbeelding

**Review by MtePoele — APPROVED** [review]

**Pull request merged**